tests/unit_tests/steps/test_archive_objects.py

```diff
@@ -77,26 +77,24 @@ def test_for_library(self, stub_tool_box,
 
         help_command = ['ar', '--version']
         fake_process.register(help_command, stdout='1.0.0')
-        ar_command = ['ar', 'cr', '/fab/proj/build_output/mylib.a',
-                      'util1.o', 'util2.o']
+        ar_command = ['ar', 'cr', '/fab/proj/build_output/mylib.a', 'util1.o', 'util2.o']
```

Comment:
Again, I didn't check with Fab's coding style, but general recommendation is 80 (https://peps.python.org/pep-0008/#maximum-line-length ... actually, 79) characters, which I think was the reason why this was split where it was.